.github/workflows/weblate-sync-pot.yml

```diff
@@ -20,6 +20,9 @@ jobs:
       - name: Pacify git's permission check
         run: git config --global --add safe.directory /__w/
 
+      - name: Install packages from extraction
```

Comment:
Since this is running in the `unit-tests` container I'd prefer to add the dependencies there.  `python3` is already available, fwiw, so just the `polib`.